tests/pytorch/test_nn.py

```diff
@@ -1801,3 +1801,22 @@ def test_BiasedMultiheadAttention(feat_size, num_heads, bias, attn_bias_type, at
     out = net(ndata, attn_bias, attn_mask)
 
     assert out.shape == (16, 100, feat_size)
+
+@pytest.mark.parametrize('feat_size', [128, 512])
+@pytest.mark.parametrize('ffn_size', [1024, 2048])
+@pytest.mark.parametrize('num_heads', [8, 16])
+@pytest.mark.parametrize('attn_bias_type', ['add', 'mul'])
+@pytest.mark.parametrize('norm_first', [True, False])
+@pytest.mark.parametrize('dropout', [0.1, 0.5])
+@pytest.mark.parametrize('activation', ['relu', 'leaky_relu'])
```

Comment:
L1805-1811 will result in 2^7 unit tests. It should be fine to test a single value for `feat_size`, `ffn_size`, `num_heads`, `dropout` and `activation`.